packages/web/src/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+
+import { Markprompt } from './Markprompt.js';
+import { type MarkpromptOptions } from './types.js';
+
+function getHTMLElement(
+  value: HTMLElement | string,
+  environment: typeof window = window,
+): HTMLElement {
+  if (typeof value !== 'string') return value;
+  const el = environment.document.querySelector<HTMLElement>(value);
+  if (!el) throw new Error(`Could not find element with selector "${value}"`);
+  return el;
+}
+
+export function markprompt(
+  /** Your Markprompt project key */
+  projectKey: string,
+  /** The element or selector to render Markprompt into */
+  container: HTMLElement | string,
+  /** Options for customizing Markprompt */
+  options?: MarkpromptOptions,
+) {
```

Comment:
To document function parameters, you need to use the `@param` tag on the function jsdoc.

```suggestion
/**
 * @param projectKey Your Markprompt project key
 * @param container The element or selector to render Markprompt into
 * @param options Options for customizing Markprompt
 */
export function markprompt(
  projectKey: string,
  container: HTMLElement | string,
  options?: MarkpromptOptions,
) {
```